src/garage/torch/algos/maml.py

```diff
@@ -150,6 +149,32 @@ def train_once(self, runner, all_samples, all_params):
 
         return average_return
 
+    def _train_value_function(self, paths):
```

Comment:
@naeioi this open the possibility of applying MAML to both the VF and the policy. are you interested in trying this?